ios/FluentUI/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -13,74 +13,67 @@ import UIKit
-        case .xSmall:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xSmall
-        case .small:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.small
-        case .medium:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.medium
-        case .large:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.large
-        case .xLarge:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xLarge
-        }
-    }
 }
 
-/// Representation of design tokens to buttons at runtime which interfaces with the Design Token System auto-generated code.
-/// Updating these properties causes the SwiftUI button to update its view automatically.
-class MSFActivityIndicatorTokens: MSFTokensBase, ObservableObject {
-    @Published public var activityIndicatorSize: CGFloat!
-    @Published public var defaultColor: UIColor!
-    @Published public var thickness: CGFloat!
-
-    var size: MSFActivityIndicatorSize {
-        didSet {
-            if oldValue != size {
-                updateForCurrentTheme()
-            }
-        }
-    }
+/// Design token set for the `ActivityIndicator` control.
+public class ActivityIndicatorTokens: ControlTokens {
+    /// Creates an instance of `ActivityIndicatorTokens` with optional token value overrides.
+    /// - Parameters:
+    ///   - size: MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    ///   - defaultColor: The default color of the Activity Indicator.
+    ///   - side: The value for the side of the square frame of an Activity Indicator.
+    ///   - thickness: The value for the thickness of the ActivityIndicator ring.
+    public init(size: MSFActivityIndicatorSize,
+                defaultColor: DynamicColor? = nil,
+                side: CGFloat? = nil,
+                thickness: CGFloat? = nil) {
 
-    init(size: MSFActivityIndicatorSize) {
         self.size = size
-
         super.init()
 
-        self.themeAware = true
-
-        updateForCurrentTheme()
+        // Optional overrides
+        if let defaultColor = defaultColor {
+            self.defaultColor = defaultColor
+        }
+        if let side = side {
+            self.side = side
+        }
+        if let thickness = thickness {
+            self.thickness = thickness
+        }
     }
 
-    override func updateForCurrentTheme() {
-        let currentTheme = theme
-        let appearanceProxy = currentTheme.MSFActivityIndicatorTokens
+    let size: MSFActivityIndicatorSize
+
+    // MARK: - Design Tokens
 
-        defaultColor = appearanceProxy.defaultColor
+    lazy var defaultColor: DynamicColor = aliasTokens.foregroundColors[.neutral4]
 
+    lazy var side: CGFloat = {
         switch size {
         case .xSmall:
-            activityIndicatorSize = appearanceProxy.size.xSmall
-            thickness = appearanceProxy.thickness.xSmall
+            return 12
```

Comment:
Seems odd that none of the floats reference global tokens?

Reply:
Agreed. Most of these values are available in GlobalTokens using `borderSize` for thickness and `iconSize` for this one). That said, this control hasn't been formally specced in the vNext system yet, so there's not a perfect 1-1 mapping.

Reply:
The context behind this is that there are no tokens defining frame size or a standard created for control sizes so far. I'm not sure if the design team will standardize in that level of detail as this is not an icon per se.
This is not different from the current definition on the old YAML file as of now, so seems out of the scope of this change.